src/pykeen/nn/norm.py

```diff
@@ -29,3 +31,27 @@ def powersum_norm(x: torch.FloatTensor, p: float, dim: Optional[int], normalize:
         return value
     dim = torch.as_tensor(x.shape[-1], dtype=torch.float, device=x.device)
     return value / dim
+
+
+def complex_normalize(x: torch.Tensor) -> torch.Tensor:
+    r"""Normalize the length of relation vectors, if the forward constraint has not been applied yet.
```

Comment:
Here the docstring might be a bit imprecise:

We normalize the complex vector *element-wise*, i.e. each element of the tensor (interpreted as a complex number) is normalized by its length (thus ensuring that is represents a rotation in complex plane). This is different to normalizing the vector to unit length.

Reply:
@mberr will you update this docstring and trigger tests please?